firebase-crashlytics-ndk/src/main/jni/libcrashlytics/include/crashlytics/external/crashlytics.h

```diff
@@ -14,85 +14,152 @@
+    typedef struct __crashlytics_unspecified                __crashlytics_unspecified_t;
 
-typedef void                            (*__crashlytics_set_user_id_internal_t)(__crashlytics_unspecified_t *, const char *);
+    typedef __crashlytics_unspecified_t*    (*__crashlytics_initialize_t)      ();
+    typedef void                            (*__crashlytics_set_internal_t)    (__crashlytics_unspecified_t *, const char *, const char *);
+    typedef void                            (*__crashlytics_log_internal_t)    (__crashlytics_unspecified_t *, const char *);
+    typedef void                            (*__crashlytics_set_user_id_internal_t)(__crashlytics_unspecified_t *, const char *);
+    typedef void                            (*__crashlytics_dispose_t)         (__crashlytics_unspecified_t *);
 
-struct  __crashlytics_context {
-/* API ---------------------------------------------------------------------------------------------------------*/
-        __crashlytics_set_t                            set;
-        __crashlytics_log_t                            log;
-        __crashlytics_set_user_id_t                    set_user_id;
-/*--------------------------------------------------------------------------------------------------------------*/
+    struct __crashlytics_context {
 
-        __crashlytics_set_internal_t                   __set;
-        __crashlytics_log_internal_t                   __log;
-        __crashlytics_set_user_id_internal_t           __set_user_id;
+        __crashlytics_set_internal_t __set;
+        __crashlytics_log_internal_t __log;
+        __crashlytics_set_user_id_internal_t __set_user_id;
 
-        __crashlytics_unspecified_t*                   __ctx;
-        __crashlytics_dispose_t                        __dispose;
-};
+        __crashlytics_unspecified_t *__ctx;
+        __crashlytics_dispose_t __dispose;
+    };
 
 #define __CRASHLYTICS_NULL_CONTEXT                             (struct __crashlytics_context *) 0
 #define __CRASHLYTICS_INITIALIZE_FAILURE                       (struct __crashlytics_unspecified *) 0
 #define __CRASHLYTICS_DECORATED                                __attribute__ ((always_inline))
 
-/* API ---------------------------------------------------------------------------------------------------------*/
 
-static inline crashlytics_context_t* crashlytics_init()                                   __CRASHLYTICS_DECORATED;
-static inline void                   crashlytics_free(crashlytics_context_t** context)    __CRASHLYTICS_DECORATED;
+    static inline __crashlytics_context_t* __crashlytics_init() __CRASHLYTICS_DECORATED;
+    static inline void                     __crashlytics_free(__crashlytics_context_t **context) __CRASHLYTICS_DECORATED;
+
+    __crashlytics_context_t *__context;
+
+    bool VerifyCrashlytics() {
+        if (__context) {
+            return true;
+        }
+        return false;
+    }
+
+    void Initialize() {
```

Comment:
All definitions within this file should be declared as inline, otherwise, you are going to run into a "multiple definition" compiler error. You can test this by including this file into two separate translation units and compiling.